railway up crashes on GitHub Actions

The latest version of the railway CLI is crashing when running `railway up` as part of a GitHub Action.

Here's how the stack trace looks in my project using Railway:
```
Laying tracks in the clouds...
🚨 Looks like something derailed, Press Enter to send error logs (^C to quit)Taking notes...
panic: runtime error: invalid memory address or nil pointer dereference [recovered]
	panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x46da9e]

goroutine 1 [running]:
sync.(*RWMutex).Lock(0xc00038[7](https://github.com/RevolutionTech/letter-joy/runs/6662744725#step:7:8)630?)
	/opt/hostedtoolcache/go/1.1[8](https://github.com/RevolutionTech/letter-joy/runs/6662744725#step:7:9).2/x64/src/sync/rwmutex.go:13[9](https://github.com/RevolutionTech/letter-joy/runs/6662744725#step:7:10) +0x1e
github.com/briandowns/spinner.(*Spinner).Stop(0xce3540)
	/home/runner/go/pkg/mod/github.com/briandowns/spinner@v1.11.1/spinner.go:327 +0x39
github.com/railwayapp/cli/ui.StopSpinner(...)
	/home/runner/work/cli/cli/ui/spinner.go:55
github.com/railwayapp/cli/cmd.(*Handler).Panic(0xc000035cf0, {0xa04688, 0xc00002c0c0}, {0xc0000224c0, 0x40}, {0xc0002aa680, 0x531}, {0x936d0f, 0x2}, {0xd13960, ...})
	/home/runner/work/cli/cli/cmd/panic.go:26 +0x194
main.contextualize.func1.1()
	/home/runner/work/cli/cli/main.go:45 +0x192
panic({0x8bedc0, 0xcd08a0})
	/opt/hostedtoolcache/go/1.18.2/x64/src/runtime/panic.go:838 +0x207
sync.(*RWMutex).Lock(0xc000133a38?)
	/opt/hostedtoolcache/go/1.18.2/x64/src/sync/rwmutex.go:139 +0x1e
github.com/briandowns/spinner.(*Spinner).Stop(0xce3540)
	/home/runner/go/pkg/mod/github.com/briandowns/spinner@v1.11.1/spinner.go:327 +0x39
github.com/railwayapp/cli/ui.StopSpinner(...)
	/home/runner/work/cli/cli/ui/spinner.go:55
github.com/railwayapp/cli/cmd.(*Handler).Up(0xc000035cf0, {0xa04688, 0xc00002c0c0}, 0xc00001ea00)
	/home/runner/work/cli/cli/cmd/up.go:[10](https://github.com/RevolutionTech/letter-joy/runs/6662744725#step:7:11)3 +0x7e5
main.contextualize.func1(0xc000[11](https://github.com/RevolutionTech/letter-joy/runs/6662744725#step:7:12)4dc0, {0xd[13](https://github.com/RevolutionTech/letter-joy/runs/6662744725#step:7:14)960, 0x0, 0x0})
	/home/runner/work/cli/cli/main.go:56 +0x17e
github.com/spf13/cobra.(*Command).execute(0xc0001[14](https://github.com/RevolutionTech/letter-joy/runs/6662744725#step:7:15)dc0, {0xd13960, 0x0, 0x0})
	/home/runner/go/pkg/mod/github.com/spf13/cobra@v1.0.0/command.go:842 +0x67c
github.com/spf13/cobra.(*Command).ExecuteC(0xcd9e60)
	/home/runner/go/pkg/mod/github.com/spf13/cobra@v1.0.0/command.go:950 +0x39c
github.com/spf13/cobra.(*Command).Execute(...)
	/home/runner/go/pkg/mod/github.com/spf13/cobra@v1.0.0/command.go:887
main.main()
	/home/runner/work/cli/cli/main.go:[34](https://github.com/RevolutionTech/letter-joy/runs/6662744725#step:7:35)0 +0x65
```

You can take a look at the failing deploy using the latest CLI (1.7.3):
https://github.com/RevolutionTech/letter-joy/runs/6662744725

I wasn't able to reproduce this crash when running `railway up` locally on my Macbook, but it happens consistently on GitHub Actions.

Workaround: [rolling the CLI back to 1.6.0](https://github.com/RevolutionTech/letter-joy/commit/bf8e8ff144a4d3fc13ce0032258c950a7bacb51d) [resolves the issue](https://github.com/RevolutionTech/letter-joy/runs/6662795532).

--- a/ui/spinner.go
+++ b/ui/spinner.go
@@ -52,5 +52,8 @@ func StopSpinner(msg string) {
 		s.FinalMSG = msg + "\n"
 	}
 
-	s.Stop()
+	// NOTE: Running Stop() when not active triggers a nil pointer
+	if s.Active() {
+		s.Stop()
+	}
 }

